.circleci/config.yml

```diff
@@ -2,26 +2,14 @@ version: 2.1
 setup: true
 
 orbs:
-  git-shallow-clone: guitarrapc/git-shallow-clone@2.0.3
-  continuation: circleci/continuation@0.1.2
-  node: circleci/node@5.0.3
-
-jobs:
-  setup:
-    executor: continuation/default
-    steps:
-      - git-shallow-clone/checkout_advanced:
-          clone_options: '--depth 1 --verbose'
-      - node/install:
-          node-version: '16.17'
-      - run:
-          name: Generate config
-          command: |
-            node ./scripts/ci/generate.js
-      - continuation/continue:
-          configuration_path: .circleci/config.generated.yml
+  path-filtering: circleci/path-filtering@0.1.3
 
 workflows:
   setup:
     jobs:
-      - setup
\ No newline at end of file
+      - path-filtering/filter:
+          name: Create dynamic jobs
+          config-path: .circleci/config.old.yml
+          base-revision: next
+          mapping: |
+            .*\.[tj]s$ run-js true
```

Comment:
This checks whether the diff against the `next` branch has a JS or TS file. 

If so, it sends `<< pipeline.parameters.run-js >>` as `true`, which causes the [sandboxes](https://github.com/kienstra/storybook/pull/2/files#diff-f5cf45c38f4759120f2b2da5ca659b97fa43e3db9e6962eb882b685a13dd3241R455) workflow to run.